docs/source/en/api/pipelines/flux.md

```diff
@@ -160,18 +162,115 @@ image.save("flux-fp8-dev.png")
 
 ## FluxPipeline
 
-[[autodoc]] FluxPipeline
-	- all
-	- __call__
+[[autodoc]] FluxPipeline - all - **call**
```

Comment:
Not sure why these were changed, please revert to keep the scope of this PR limited

Reply:
sure will do